Jenkinsfile

```diff
@@ -3,9 +3,11 @@
  https://github.com/jenkins-infra/pipeline-library/
 */
 buildPlugin(
+ forkCount: '1C', // run this number of tests in parallel for faster feedback.  If the number terminates with a 'C', the value will be multiplied by the number of available CPU cores
   useContainerAgent: true, // Set to `false` if you need to use Docker for containerized tests
   configurations: [
     [platform: 'linux', jdk: 11],
     [platform: 'linux', jdk: 17],
     [platform: 'linux', jdk: 21],
+    [platform: 'windows', jdk: 17],
```

Comment:
Windows doesn't make any sense here. It's a plugin that will work only on Kubernetes linux nodes